src/distributed/redis/subservice.ts

```diff
@@ -13,13 +13,15 @@ export class RedisSubservice implements DistributedSubservice {
   private sub!: Redis
   private callbacks: { [channel: string]: (message: any) => Promise<void> } = {}
   private pings!: PingPong
+  private scope!: string
 
   constructor(private config: RedisConfig) {}
 
   async setup() {
     this.nodeId = Math.round(Math.random() * 1000000)
     this.logger.info(`Id is ${clc.bold(this.nodeId)}`)
 
+    this.scope = process.env.REDIS_SCOPE || this.config.scope
```

Comment:
You forgot to use the redis.scope setting in the .json file so I added this here